components/icon/stories/icon.stories.js

```diff
@@ -169,3 +170,161 @@ const TestTemplate = (args) => html`
 		)}
 	</div>
 `;
+
+/**
+ * Display all icon sizes for the Docs page.
+ */
+const WorkflowSizingTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			"display": "flex",
+			"gap": "24px",
+			"flexWrap": "wrap",
+		})}
+	>
+		${["xs","s","m","l","xl"].map(
+			(size) => html`
+				<div
+					style=${styleMap({
+						"display": "flex",
+						"gap": "16px",
+						"flexDirection": "column",
+						"alignItems": "center",
+						"flexBasis": "80px",
+					})}
+				>
+					${Typography({
+						semantics: "detail",
+						size: "s",
+						content: [
+							{
+								xs: "Extra-small",
+								s: "Small",
+								m: "Medium",
+								l: "Large",
+								xl: "Extra-large",
+							}[size],
+						],
+						customStyles: {
+							"white-space": "nowrap",
+							"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+						}
+					})}
+					${Template({ ...args, size })}
+				</div>
+			`
+		)}
+	</div>
+`;
+
+
+/**
+ * Helper template function to display multiple icons.
+ */
+const IconListTemplate = (args, iconSet = "workflow", iconsList = []) => html`
+	<div
+		style=${styleMap({
+			"display": "flex",
+			"gap": "32px",
+			"flexWrap": "wrap",
+		})}
+	>
+		${iconsList.map(
+			(iconName) => Template({
+				...args,
+				iconSet,
+				iconName,
+			})
+		)}
+	</div>
+`;
+
+/**
+ * Display examples of multiple workflow icons on the Docs page.
+ */
+const WorkflowDefaultTemplate = (args) => html`
+	${IconListTemplate(
+		args, 
+		"workflow", 
+		[
+			"Alert",
+			"Asset",
+			"Actions",
+			"ArrowDown",
+			"Camera",
+			"Copy",
+			"DeviceDesktop",
+			"Download",
+			"FilterAdd",
+			"Form",
+			"Light",
+			"Polygon",
+		]
+	)}
+`;
+
+/**
+ * Display examples of multiple UI icons on the Docs page.
+ */
+const UIDefaultTemplate = (args) => html`
```

Comment:
Would it be worth showing more UI icons in this story or no?

Reply:
I could see it being useful if there's an example with a different number of sizes than currently shown! I also wonder if it could be helpful to users if we called out the size of the icons in this example 🤔 I'm a bit torn and could go either way.

Reply:
I had the same thought, it could be nice to see the size of the icons or the classname or something, but I can also see how adding that could contribute a bit too much noise.

I think showing the two UI icons here is ok, if there's an example that has just 1 or 2 sizes, that might be nice to show too!